### PR TITLE
feat(kuma-cp) pick service type and add custom annotations

### LINKED ISSUE #960: HELM - add option to not expose services + add annotations
### Summary

Some of our clients would like to keep services as `ClusterIP` not `NodePort` or `LoadBalancer`. Also there is need to add custom annotations for internal loadbalancers.

### PR DESCRIPTION
**Pull request by jakubdyszkiewicz**
### Summary

Fix #960

Instead of useNodePort you can now specify type. Additionaly you can add your own annotations.

### CODE REVIEW

File: deployments/charts/kuma/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: kuma
 description: A Helm chart for the Kuma Control Plane
 type: application
-version: 0.1.0
+version: 0.2.0
```
Comment (nickolaev) [inline deployments/charts/kuma/Chart.yaml:5]: 👍

File: deployments/charts/kuma/templates/cp-global-sync-service.yaml [hunk 2]
```diff
@@ -2,17 +2,19 @@
...
+    {{- end }}
 spec:
-  {{- if .Values.controlPlane.useNodePort }}
-  type: NodePort
-  {{- else }}
-  type: LoadBalancer
-  {{- end }}
+  type: {{ .Values.controlPlane.globalRemoteSyncService.type }}
   ports:
     - port: 5685
```
Comment (nickolaev) [inline deployments/charts/kuma/templates/cp-global-sync-service.yaml:14]: Shall we parametrize this port too? Just in can and while we are here.

Comment (jakubdyszkiewicz) [reply deployments/charts/kuma/templates/cp-global-sync-service.yaml]: done

**Comment by jakubdyszkiewicz** [comment]
tested E2E on my machine, not waiting on CI

**Pull request merged**